src/App.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { Link, Route, Routes } from 'react-router-dom';
+import './styles/style.scss';
+import Books from './combiner/Books';
+import Cartegory from './components/Category';
+
+export default function App() {
+  return (
+    <>
+      <nav className="navbar">
+        <div className="Title-name">
+          BOOK STORE
+        </div>
+        <div className="nav-links">
+          <Link to="/Books"> Books </Link>
+          <Link to="/Category"> Category </Link>
+        </div>
+      </nav>
```

Comment:
[ Optional Changes ]

- [ ] You could have moved this to the Header.js component to serve as your navbar and then import and use that Header component here. That makes your App.js lean and clean. 👍🏽

Reply:
Its Done thanks for pointing it out.